spec/ssh_scan/banner_spec.rb

```diff
@@ -84,6 +93,9 @@
       "SSH-2.0-ros" => {
         :os_class => SSHScan::OS::ROS,
       },
+      "SSH-1.99-DOPRA" => {
```

Comment:
@claudijd Since this was pretty small, I changed this in the recent PR where I modularize tests.